newIDE/app/src/ObjectsList/index.js

```diff
@@ -261,15 +261,15 @@ export default class ObjectsListContainer extends React.Component<
 
   _duplicateObject = (objectWithContext: ObjectWithContext) => {
     this._copyObject(objectWithContext);
-    this._editName(this._paste(objectWithContext));
+    this._pasteAndRename(objectWithContext);
   };
 
-  _pasteObject = (objectWithContext: ObjectWithContext) => {
+  _pasteAndRename = (objectWithContext: ObjectWithContext) => {
     this._editName(this._paste(objectWithContext));
   };
 
-  _paste = (objectWithContext: ObjectWithContext): ObjectWithContext => {
-    if (!Clipboard.has(CLIPBOARD_KIND)) return;
+  _paste = (objectWithContext: ObjectWithContext): ?ObjectWithContext => {
```

Comment:
Small change, the return type is *optionally* a `ObjectWithContext` (hence the "?"). null can be returned in can nothing is pasted. Flow help us to verify that anything calling _paste will support getting null (and it's not the case, it will give us errors)